README.md

```diff
@@ -281,7 +281,7 @@ For authentication you can also use Travis CI secure environment variable:
 * **password**: PyPI Password.
 * **server**: Optional. Only required if you want to release to a different index. Follows the form of 'https://mypackageindex.com/index'. Defaults to 'https://pypi.python.org/pypi'.
 * **distributions**: Optional. A space-separated list of distributions to be uploaded to PyPI. Defaults to 'sdist'.
-* **skip_upload_docs**: Optional. When set to `true`, documentation is not uploaded. Defaults to `false`.
+* **skip_upload_docs**: Optional. When set to `false`, documentation is uploaded. Defaults to `true`.
```

Comment:
Perhaps it is worth mentioning that at the moment setting this to `false` will result in a build failure for PyPI? Otherwise someone might set it to `false` and file an error report.